src/coreclr/jit/emit.h

```diff
@@ -628,8 +628,8 @@ class emitter
 #define MAX_ENCODED_SIZE 15
 #elif defined(TARGET_ARM64)
 #define INSTR_ENCODED_SIZE 4
-        static_assert_no_msg(INS_count <= 512);
-        instruction _idIns : 9;
+        static_assert_no_msg(INS_count <= 1024);
+        instruction _idIns : 10;
```

Comment:
I added a bit here since we now have more than 512 instructions (at that for sure will be increasing in future) but I also removed an used (on arm) `_idCallRegPtr` bit so the total size of the struct remains the same - 16b